WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -31,6 +31,22 @@ struct HubMenu: View {
                 .onAppear {
                     viewModel.setupMenuElements()
                 }
+                .fullScreenCover(isPresented: $viewModel.showsPOS) {
+                    if let cardPresentPaymentService = viewModel.cardPresentPaymentService,
+                       let orderService = POSOrderService(siteID: viewModel.siteID,
+                                                          credentials: viewModel.credentials) {
+                        PointOfSaleEntryPointView(
+                            itemsController: PointOfSaleItemsController(itemProvider: viewModel.posItemProvider),
+                            onPointOfSaleModeActiveStateChange: { isEnabled in
+                                viewModel.updateDefaultConfigurationForPointOfSale(isEnabled)
```

Comment:
`updateDefaultConfigurationForPointOfSale` calls `updateTabBarVisibility` which may not be needed as we present a full screen cover